WooCommerce/Classes/ViewRelated/MainTabBarController.swift

```diff
@@ -183,13 +183,25 @@ final class MainTabBarController: UITabBarController {
     /// Switches the TabBarController to the specified Tab
     ///
     func navigateTo(_ tab: WooTab, animated: Bool = false, completion: (() -> Void)? = nil) {
+        navigateToTabWithNavigationController(tab, animated: animated) { navigationController in
```

Comment:
Nit: Should we replace `navigationController` with an underscore since it's not used in the closure?

Reply:
Good catch! updated in https://github.com/woocommerce/woocommerce-ios/pull/10119/commits/84869b16c12c4c0a0e3f0db5986260f0db27ca42